modules/nf-core/islandpath/tests/main.nf.test

```diff
@@ -0,0 +1,48 @@
+// nf-core modules test islandpath
+nextflow_process {
+
+    name "Test Process ISLANDPATH"
+    script "../main.nf"
+    process "ISLANDPATH"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gunzip"
+    tag "islandpath"
+
+    test("bacteroides_fragilis - gbff") {
+
+        setup {
+            run("GUNZIP") {
+                script "../../gunzip/main.nf"
+                process {
+                    """
+                    input[0] = Channel.fromList([
+                                    tuple([ id:'test' ], // meta map
+                                    file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/genome/genome.gbff.gz', checkIfExists: true)),
+                                ])
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = GUNZIP.out.gunzip
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out[0][0]).match() },
```

Comment:
can you please do .gff instead of [0][0] here? Conda might work with it as well..

Reply:
Done in 78d5994b21d2d58ee604848777b636bd8c09a239